src/components/discover-sheet/DiscoverSearch.js

```diff
@@ -250,6 +250,7 @@ export default function DiscoverSearch() {
       <SearchContainer>
         <CurrencySelectionList
           footerSpacer
+          fromDiscover
```

Comment:
curious to know why this was needed

Reply:
+1,I see what you're trying to do but iirc this prop is for use in swaps and behaves a lil diff than you expect, may want to check currentRoute or pass a different prop here

Reply:
after looking a lil deeper, i'm totally fine with this 😆